integration/tests/base.py

```diff
@@ -6,46 +6,50 @@
 from utils.faucet import Faucet
 from utils.operator import Operator
 from utils.solana_client import SolanaClient
-from utils.web3client import NeonChainWeb3Client
+from utils.web3client import NeonChainWeb3Client, Web3Client
 
 
 class BaseTests:
     acc: eth_account.signers.local.LocalAccount
     operator: Operator
     faucet: Faucet
     web3_client: NeonChainWeb3Client
+    web3_client_sol: Web3Client
     sol_client: SolanaClient
     sol_price: float
 
     @pytest.fixture(autouse=True)
-    def prepare(self, operator: Operator, faucet: Faucet, web3_client, sol_client):
+    def prepare(self, operator: Operator, faucet: Faucet, web3_client, web3_client_sol, sol_client):
```

Comment:
Let’s create an issue for refactoring this base class because we have a mess right now.
1. several web clients but in general we are using one (for neon)
2. create_tx_object and similar methods

Reply:
https://neonlabs.atlassian.net/browse/NDEV-2384